test/functional/test_framework/script_util.py

```diff
@@ -5,14 +5,14 @@
 """Useful Script constants and utils."""
 from test_framework.script import (
     CScript,
-    hash160,
-    sha256,
     OP_0,
-    OP_DUP,
-    OP_HASH160,
     OP_CHECKSIG,
+    OP_DUP,
     OP_EQUAL,
     OP_EQUALVERIFY,
+    OP_HASH160,
+    hash160,
+    sha256,
```

Comment:
Any specific reason for this refactoring?

Reply:
AFAIK, we follow the common practice of keeping the Python imports sorted in lexicographical order. I don't know though if we have written that down anywhere in our `doc`s as guidelines.